docs/src/getting-started/cli.md

```diff
@@ -0,0 +1,204 @@
+# CLI configuration and usage
+
+This guide shows the available commands and their configuration options to run with the Miden Faucet CLI.
+
+## Available Commands
+
+| Command | Description |
+|---------|-------------|
+| `start` | Start the faucet server |
+| `create-faucet-account` | Create a new faucet account |
+| `create-api-keys` | Generate API keys for authentication |
+| `help` | Show help information |
+
+## Configuration Methods
+
+The Miden Faucet can be configured using:
+
+1. **Command-line arguments**
+2. **Environment variables**
+
+## Command-Line Arguments
+
+### Basic Configuration
+
+```bash
+miden-faucet start \
+  --endpoint <URL> \
+  --node-url <URL> \
+  --account <PATH> \
+  --network <NETWORK>
+```
+
+### All Available Options
+
+| Option | Description | Default | Required |
+|--------|-------------|---------|----------|
+| `--endpoint` | Faucet endpoint (IP:PORT) | - | Yes |
+| `--node-url` | Miden node RPC endpoint | - | Yes |
+| `--account` | Path to faucet account file | - | Yes |
+| `--network` | Network configuration | `localhost` | No |
+| `--timeout` | RPC request timeout | `5s` | No |
+| `--max-claimable-amount` | Max claimable base units per request | `1000000000` | No |
+| `--store` | SQLite store path | `faucet_client_store.sqlite3` | No |
```

Comment:
Do we also now need to specify explorer URL? (if so, this would also be relevant in a few other places in the book).